tests/operator/operator.go

```diff
@@ -3442,6 +3424,18 @@ func nodeSelectorExistInDeployment(virtClient kubecli.KubevirtClient, deployment
 	return true
 }
 
+func sanityCheckDeploymentsExist(virtClient kubecli.KubevirtClient, namespace string) {
+	Eventually(func() error {
```

Comment:
there is no need to expose these arguments as `kubevirt.Client()` returns a singleton, and the `namespace` is specific and constant to this test.

Reply:
Right, I'll improve it.